0001-typedb/01-start/04-iam-schema.md

```diff
@@ -0,0 +1,302 @@
+under a single management framework.
+
+<div class="note">
+[Note]
+The terminology used in the IAM [schema](03-quickstart.md#prepare-a-tql-file) is defined according to the sources 
+mentioned above and ISO/IEC 15408-1 standard within the context of an information system.
+</div>
+
+<div class="note">
+[Note]
+The Identity and Access Management database [schema](03-quickstart.md#prepare-a-tql-file) and accompanying 
+[dataset](03-quickstart.md#insert-data) described here are used in most examples throughout TypeDB documentation, 
+including the [Quickstart guide](03-quickstart.md).
+</div>
+
+## Overview
+
+As with any [TypeDB schema](../02-dev/02-schema.md), the IAM schema consists of:
+
+- Types
+  - Entity types
+  - Relation types
+  - Attribute types
+- Rules
+
+The [schema](../02-dev/02-schema.md) page will later provide more information to better understand the 
+terminology of TypeDB schemas.
+
+The main entity types in the IAM schema:
+
+- Subject — Active entity in the _system_ that performs operations on objects. 
+  - User — Human or IT entity possibly interacting with the _system_ from outside of the _system_ boundary.
+    - Person — User with `credentials`, `email`, and `full-name` attributes.
+  - User-group — A group of users that share the same role.
+- Object — Passive entity in the _system_, that contains or receives information, and upon which subjects perform
+  operations. 
+  - Resource — Anything usable or consumable in the _system_.
+    - File — File in some filesystem with `path` (to the file) and `size` (in KB) attributes.
+  - Resource-collection — A collection of resources on which the same operations can be performed.
+- Action — An operation or operation set that can be performed on a specific type of object. Its name is stored in
+  the `action-name` attribute.
+
+<div class="note">
+[Note]
+In the list above some entities have attributes that are mentioned in their description.
+</div>
+
+The main relation types in the IAM schema:
+
+- Permission — A relation that relates a subject to a specific access.
+- Access — A relation that relates an object to a valid action performed on it.
+
+The types mentioned above combined together in a schema will look like this:
+
+![IAM core schema](../../images/iam/simplified.png)
+
+The illustration above is not the full IAM schema but only the most important part of it. The least important parts
+have been omitted from the image to reduce the complexity of the schema. For a full version, please see the image in
+the [Full schema](#full-schema) section below.
+
+## Permission and Access relations
+
+One of the core elements of the IAM schema is the relations between `subject` and `object` entity types that allow  
+to set `permissions` to access something.
+
+There are two relation types involved:
+
+- `permission`
+- `access`
+
+The `permission` relation connects a `subject` (e.g. `person`) via a `permitted-subject` role and `access` relation via 
```

Comment:
All role names have been shortened to one-word as per IAM code review. Please see schema in IAM project. I would also potentially mention accesses before permissions and explain why they exist (to define what operations can be performed on each object). Otherwise it might feel a bit confusing.